init/module-setup.sh

```diff
@@ -25,6 +25,11 @@ kdump_check_net() {
     elif [ "${KDUMP_NETCONFIG%:force}" != "$KDUMP_NETCONFIG" ]; then
         # always set up network
         kdump_neednet=y
+    elif [ -f "/usr/lib/fence_kdump_send" ] &&
+         ( [[ $KDUMP_PRESCRIPT =~ "fence_kdump_send" ]] || \
```

Comment:
It makes no sense to use fence_kdump_send as a prescript. It would tell the fencing node that this node has been shot down. The fencing node will probably use a real stonith device (ipmi, apc, nova, …) and prevent kdump from finishing.

In short, this check seems unnecessary, because the configuration is wrong.

Reply:
My mistake. Remove the unnecessary check of PRESCRIPT in the new commit.